simbad/command_line/simbad_full.py

```diff
@@ -44,7 +44,7 @@ def main():
     args = simbad_argparse().parse_args()
 
     args.work_dir = simbad.command_line.get_work_dir(
-        args.run_dir, work_dir=args.work_dir, ccp4_jobid=args.ccp4_jobid
+        args.run_dir, work_dir=args.work_dir, ccp4_jobid=args.ccp4_jobid, ccp4i2_xml=args.ccp4i2_xml
```

Comment:
Are you sure you meant to do this?

Reply:
Yes, this option was missing from this function in `simbad.command_line.simbad_full` and `simbad.command_line.simbad_morda` so any future ccp4i2 update where the morda search was added in would break.